fraction.rb

```diff
@@ -1,64 +1,51 @@
 class Fraction
   def initialize(numerator, denominator)
-    if ! numerator.is_a?(Integer) && denominator.is_a?(Integer)
-      # TODO: Proper execption for this issue
-      # TODO: Handle case when one or both parameters are not int but could be easily transformed to int
-      raise 'these are supposed to be numbers: '+numerator.inspect+' '+ denominator.inspect
-    end
     @numerator = numerator
     @denominator = denominator
   end
-
   def numerator
     @numerator
   end
-
   def denominator
     @denominator
   end
-
-  def simplify
-
-=begin
-     if @numerator > @denominator then b=@denominator else b=@numerator
+  def simple
```

Comment:
Do you think it is worked properly? Run something like `Fraction.new(-4, 6).simple`. It should work not only _natural_ numbers, but _integer_ ones too.